debian/changelog

```diff
@@ -1,7 +1,70 @@
-ubuntu-advantage-tools (27.2.2~21.10) impish; urgency=medium
+ubuntu-advantage-tools (27.3~21.10.1) impish; urgency=medium
 
   * d/tools.postinst:
-    - Do not fail in postinst if cloud-init did not run. (LP: #1936833)
+    - handle any cloud-id error and treat as "none" cloud
+    - purge old ua-messaging.timer/service
+  * systemd:
+    - add new ua-timer.timer that runs every 2 hours
+    - add new ua-license_check.timer that runs every 5 minutes only if
```

Comment:
Should not we state here that we are removing the existing `ua-messaging` timer as well? So it's clear that we are not keeping those two plus the existing one